cli/src/base.ts

```diff
@@ -493,4 +506,43 @@ export abstract class BaseCommand extends Command {
       this.warn(`  [${error.code}] ${error.message} at "${error.path.join('.')}"`);
     }
   }
+
+  async prompt<name extends string>(
+    options: prompts.PromptObject<name>,
+    flagValue?: boolean | string
+  ): Promise<prompts.Answers<name>> {
+    // If there's a flag, use the value of the flag
+    if (flagValue != null) return { [String(options.name)]: flagValue } as prompts.Answers<name>;
+
+    const { flags } = await this.parse(
+      { strict: false, flags: { ...BaseCommand.noInputFlag, force: Flags.boolean() } },
```

Comment:
```suggestion
      { strict: false, flags: { ...BaseCommand.noInputFlag, ...BaseCommand.forceFlag() } },
```
Otherwise -f char won't work, right?